scripts/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "workflow",
   "version": "1.0.0",
   "description": "",
+  "type": "module",
```

Comment:
why add this?

Reply:
scripts is considered as ESM by default

Reply:
You could use Top level await in ESM. IMHO: Considering `.js` as ESM by default would bring up better DX.